src/WCSimConstructPMT.cc

```diff
@@ -38,11 +38,13 @@ G4LogicalVolume* WCSimDetectorConstruction::ConstructPMT(G4String PMTName, G4Str
   
   G4double expose;
   G4double radius;
+  G4double glassthickness;
   
   WCSimPMTObject *PMT = GetPMTPointer(CollectionName);
   expose = PMT->GetExposeHeight();
   radius = PMT->GetRadius();
-  
+  glassthickness = PMT->GetPMTGlassThickness();
+
```

Comment:
It's a minor thing.  But, let's try to stick to our capitalization convention and make this glassThickness.